deploy/deployment/kubernetes/combined.yaml

```diff
@@ -88,6 +88,19 @@ spec:
                 description: Workspace defines configuration options related to how
                   DevWorkspaces are managed
                 properties:
+                  StorageClassSize:
+                    additionalProperties:
+                      anyOf:
+                      - type: integer
+                      - type: string
+                      pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                      x-kubernetes-int-or-string: true
+                    description: 'StorageClassSize defines an optional map of sizes
+                      of persistent volume claims for each storage class used by DevWorkspaces.
+                      The available keys to the map are: "common", "per-workspace".
```

Comment:
I'm not sure why these files don't have the mention of `async` using the `common` storage class PVC size that I added?

Reply:
The CI is also complaining about this :P

Reply:
You have to run `make generate` to regenerate CRDs after changing the go definition. I use
```
make generate manifests fmt generate_default_deployment generate_olm_bundle_yaml
```
to make sure all autogenerated files are in sync.